manage_user_page.php

```diff
@@ -379,3 +386,33 @@
 </div>
 <?php
 layout_page_end();
+
+
+/**
+ *
+ */
+function draw_search_section($p_user_count,$p_per_page,$p_page_count, $p_page_number, $p_filter_string) {
```

Comment:
You introduce a new function that is quite similar to existing code.
The goal should be to replace existing code by a call to the new function.

Reply:
I've copied the existent code to create the function, OK if existent code can be replaced => two calls to this function. Please let me know how I've to proceed in order to provide work as will be better to be integrated on stable code base